src/services/api/fragmentTypes.json

```diff
@@ -932,18 +953,15 @@
         "__typename": "__Type"
       },
       {
-        "kind": "INTERFACE",
+        "kind": "OBJECT",
         "name": "SahkopostiVastaanottaja",
-        "possibleTypes": [
-          {
-            "name": "KuntaVastaanottaja",
-            "__typename": "__Type"
-          },
-          {
-            "name": "ViranomaisVastaanottaja",
-            "__typename": "__Type"
-          }
-        ],
+        "possibleTypes": null,
+        "__typename": "__Type"
+      },
+      {
+        "kind": "INPUT_OBJECT",
+        "name": "SahkopostiVastaanottajaInput",
+        "possibleTypes": null,
         "__typename": "__Type"
       },
       {
```

Comment:
Tää oli muuttunut jo aiemmin oikeasti, mutta kukaan ei ollut ajanut full BE-deployta missään vaiheessa.